### PR TITLE
add Alabama and Alaska

### PR DESCRIPTION
**Pull request by jacobi973**

### CODE REVIEW

File: src/alabama/readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Jordan Scrapes Secretary of States: Alabama
+
+This package has some sample code getting data from the [Alabama Secretary of Search Business Search](https://www.sos.alabama.gov/government-records/business-entity-records). 
+
+A simple search and scrape of Alabama businesses using *FIX* incrementing ids to get the most recently registered businesses.
```
Comment (aarmora) [inline src/alabama/readme.md:5]: *FIX* ?

File: src/alaska/readme.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Jordan Scrapes Secretary of States: Alaska
+
+This package has some sample code getting data from the [Alaska Secretary of Search Business Search](https://www.commerce.alaska.gov/cbp/main/). 
+
+A simple search and scrape of Alaska businesses using *FIX* incrementing ids to get the most recently registered businesses.
```
Comment (aarmora) [inline src/alaska/readme.md:5]: *FIX*?

Comment (jacobi973) [reply src/alaska/readme.md:5]: That's strange that the FIX is in there. I Just pushed the revised verison

File: readme.md [hunk 3]
```diff
@@ -32,4 +32,8 @@
 
 [North Carolina](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/northCarolina)
 
-[Hawaii](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/hawaii)
\ No newline at end of file
+[Hawaii](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/hawaii)
+
+[Alaska](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/alaska)
+
+[Alabama](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/hawaii)
```
Comment (aarmora) [inline readme.md:39]:
```suggestion
[Alabama](https://github.com/aarmora/jordan-scrapes-secretary-of-states/tree/master/src/alabama)
```

File: src/alaska/readme.md (hunk 2 above)
Comment (aarmora) [inline src/alaska/readme.md:5]:
```suggestion
A simple search and scrape of Alaska businesses using incrementing ids to get the most recently registered businesses.
```

**Pull request merged**